screwdriver/screwdriver.go

```diff
@@ -197,7 +197,7 @@ type Event struct {
 	ID            int                    `json:"id"`
 	Meta          map[string]interface{} `json:"meta"`
 	ParentEventID int                    `json:"parentEventId"`
-	Creator       map[string]string      `json:"creator"`
+	Creator       map[string]interface{} `json:"creator"`
```

Comment:
```suggestion
	Creator       map[string]string `json:"creator"`
```

Per https://github.com/screwdriver-cd/screwdriver/issues/2892#issuecomment-1602452632, we need to change the data-schema to normalize to a standard data type instead of bleeding SCM specific data types.

References:
https://github.com/screwdriver-cd/data-schema/pull/518/files#r1238379937
https://github.com/screwdriver-cd/scm-github/pull/216/files#r1238380796